symengine/polys/uintpoly_flint.h

```diff
@@ -125,9 +125,9 @@ class UFlintPoly : public BaseType<Container, Poly>
         return r_iterator(this->template rcp_from_this_cast<Poly>(), -1);
     }
 
-    unsigned int size() const
+    int size() const
```

Comment:
I don't understand the purpose of having two functions `degree` and `size`, but if it works, okay. cc @srajangarg